test/unit_test/cb_las_spanner_test.cc

```diff
@@ -811,4 +837,76 @@ BOOST_AUTO_TEST_CASE(check_spanner_with_actions_that_are_linear_combinations_of_
   }
 }
 
+BOOST_AUTO_TEST_CASE(check_singular_value_sum_diff_for_diff_ranks_is_small)
```

Comment:
this checks the scaling factor that was added in this PR, without this the diff would be > 500

Reply:
this effects the non_degenerate singular value count if the first singular values can be huge without the scaling